packages/framework/src/Component/Image/ImageFacade.php

```diff
@@ -268,6 +268,28 @@ public function getImageUrl(DomainConfig $domainConfig, $imageOrEntity, $sizeNam
         throw new \Shopsys\FrameworkBundle\Component\Image\Exception\ImageNotFoundException();
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
+     * @param int $id
+     * @param string $extension
+     * @param string $entityName
+     * @param string|null $type
+     * @param string|null $sizeName
+     * @return string
+     */
+    public function getImageUrlNotUsingImageOrEntity(
```

Comment:
I know you already changed the method name, but it is still strange for me... I don't have any better unfortunately :-( And right now it is inconsistent with `ImageLocator`

maybe `getImageUrlFromParameters`? maybe the original `getImageUrlFromAttributes` was ok too

Reply:
oh, I forget to rename it in `ImageLocator`...ok, renamed back to `...FromAttributes`

Reply:
thanks and sorry for changing opinions during two days